kindlecomicconverter/utils.py

```diff
@@ -0,0 +1,10 @@
+import os
+import subprocess
+
+def run_subprocess_silent(command, **kwargs):
+    if (os.name == 'nt'):
+        startupinfo = subprocess.STARTUPINFO()
+        startupinfo.dwFlags |= subprocess.STARTF_USESHOWWINDOW
+        startupinfo.wShowWindow = subprocess.SW_HIDE
+        kwargs.setdefault('startupinfo', startupinfo)
```

Comment:
Is there a particular reason to use the `startupinfo` arg instead of `creationflags=subprocess.CREATE_NO_WINDOW`
https://docs.python.org/3/library/subprocess.html#subprocess.CREATE_NO_WINDOW

Reply:
You're right, I forgot we are checking for the OS.